roles/container-engine/cri-o/molecule/default/molecule.yml

```diff
@@ -13,12 +13,6 @@ platforms:
     memory: 1024
     groups:
       - kube_control_plane
-  - name: centos7
-    box: centos/7
-    cpus: 2
-    memory: 1024
-    groups:
-      - kube_control_plane
```

Comment:
Is the reason why here moves centos7 molecule test due to unstable test?
It is fine to drop the molecule test for me, just confirming.

Reply:
hmm, most pull requests are facing the same issue.
It is fine to drop the test at this time.
If necessary, we just need to revert the test later.

/lgtm

Reply:
Sorry didn't see it before, yes my idea was to remove it so test are passing and see later on if we need to add it back 😊